main.go

```diff
@@ -377,7 +372,9 @@ func doRun(filePath string) error {
 // navLeft navigates to the next item to the left
 func navLeft(ctx context.Context) error {
 	chromedp.KeyEvent(kb.ArrowLeft).Do(ctx)
-	chromedp.WaitReady("body", chromedp.ByQuery)
+	// Could wait for the location to change instead of this Sleep.
+	time.Sleep(200 * time.Millisecond)
```

Comment:
Remove for now please.

Reply:
Sorry, that's was not meant to be committed. (navLeft breaks for me locally).
Reverted it.